.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,52 @@
+name: Build and Test
+
+on:
+  push:
+  pull_request:
+    branches:
+    - develop
+
+env:
+  MAJOR: 12
+  MINOR: 1
+  RUN: ${{ github.run_number }}
+
+jobs:
+
+  build:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v2
+
+      - name: Restore dependencies
+        run: |
+          nuget sources add -username Open-Systems-Pharmacology -password ${{ secrets.GITHUB_TOKEN }} -name OSP-GitHub-Packages -source "https://nuget.pkg.github.com/Open-Systems-Pharmacology/index.json"
+          nuget sources add -name bddhelper -source https://ci.appveyor.com/nuget/ospsuite-bddhelper 
+          nuget sources add -name utility -source https://ci.appveyor.com/nuget/ospsuite-utility 
+          nuget sources add -name serializer -source https://ci.appveyor.com/nuget/ospsuite-serializer 
+          nuget sources add -name databinding -source https://ci.appveyor.com/nuget/ospsuite-databinding 
+          nuget sources add -name texreporting -source https://ci.appveyor.com/nuget/ospsuite-texreporting 
+          nuget sources add -name databinding-devexpress -source https://ci.appveyor.com/nuget/ospsuite-databinding-devexpress 
+          dotnet restore
+
+      - name: define env variables
+        run: |
+          echo "APP_VERSION=${{ env.MAJOR }}.${{ env.MINOR }}.9${{ env.RUN }}" | Out-File -FilePath $env:GITHUB_ENV -Append
```

Comment:
What is the 9 in `9${{ env.RUN }}` ?

Reply:
That just makes the version number 12.1.91234 to march the semantic requirements of the versioning when building

Reply:
I don't get it

Reply:
Why is `12.1.<run_number>` not sufficient?

Reply:
In fact, we can hard code this number. We don't even need any artifacts from this build. It's only to run tests and upload the test results as artifact